content/2020/09-28-new-file-request-apis-available.md

```diff
@@ -0,0 +1,53 @@
+---
+applied_at: "2020-09-28"
+applies_to: 
+- api
+is_impactful: false
+is_new_feature: true
+collapse: false
+show_excerpt: true
+release_source_url: ''
+---
+
+# New File Request APIs available
+
+A new collection of APIs are now available that allows
+developers to create and update File Requests. We've
+updated the [reference documentation][copy] and added
+[new guides][guides] to help you manage your file requests.
+
+To create a copy of an existing file request, all you need is 
+the unique ID of an existing file request, and the ID of the
+folder to apply the new request to.
+
+```curl
+curl -i -X POST "https://api.box.com/2.0/file_requests/42037322/copy" \
+     -H "Authorization: Bearer <ACCESS_TOKEN>" \
+     -d '{
+       "folder": {
+         "id": "2233212"
+       }       
+     }'
+```
+
+Additional APIs are available that allow a developer to get more details 
+about a file request, update a file request, and delete a file request.
+For more details, please check out the developer documentation.
+
+* **Create a template file request**: [Guide][g_template]
+* **Create a copy of a file request**: [Guide][g_copy] | [API Reference][copy]
+* **Get information about a file request**: [Guide][g_get] | [API Reference][get]
+* **Update a file request's configuration**: [Guide][g_del] | [API Reference][put]
```

Comment:
reference to guide on put and delete are flipped.

Reply:
Ah yes, nice catch